Assignment6.Rmd

```diff
@@ -62,8 +63,14 @@ M2$predict1 <- predict(c.tree1, M2, type = "class")
 
 M2$predict2 <- predict(c.tree2, M2, type = "class")
 
-table(M2$certified, M2$predict1)
+t1 <- table(M2$certified, M2$predict1)
+
+t2 <- table(M2$certified, M2$predict2)
 
-table(M2$certified, M2$predict2)
+acc1 <- sum(t1[1,1], t1[2,2])/sum(t1)
+acc1
+acc2 <- sum(t2[1,1], t2[2,2])/sum(t2)
+acc2
 
+#The pruned tree has a lower error rate as 53% of its certification predictions are accurate compared to the original tree(21%).
```

Comment:
Excellent interpretation, what do you think this means in terms of overfitting though?